test/mini_racer_test.rb

```diff
@@ -107,7 +108,7 @@ def test_it_can_timeout_during_serialization
     assert_raises(MiniRacer::ScriptTerminatedError) do
       context.eval 'var a = {get a(){ while(true); }}; a'
     end
-  end
+  end unless defined?(TruffleRuby) #&& !Polyglot::InnerContext.instance_methods.include?(:stop)
```

Comment:
can be removed now

Reply:
This has been moved to a skip and it depends on the truffle exception issue that was filed.